### ISSUE TITLE
@import continuation lines

### ISSUE THREAD

**Issue by ggrothendieck** [r-lib/roxygen2#1090]
If we have a package with this single R file

```
#' @import zoo

#' Different name for calling zoo.
#'
#' @params ... passed to zoo.
#' @return zoo object.
#'
#' @export 
zoo2 <- function(...) zoo(...)
```

then running document() on it gives this message
```
zoo2.Rd is missing name/title. Skipping 
```

If I put the @import at the bottom of the file  or put some dummy code line after the @import line then it works as expected not producing that warning. It seems it is regarding the line beginning with Different as being continued from the @import line.

I didn't notice any documentation that would lead one to understand that that is how it works.  It was quite unexpected and took some time to realize what was going on.

**Comment by gaborcsardi** [comment]
This is not a bug, empty lines are ignored by roxygen2, so as far as it is concerned, this is just one documentation block. Indeed, if you want to create two blocks, then you need to do some code lines between them. People usually do this:

```r
#' @import zoo
NULL
```

**Comment by ggrothendieck** [comment]
I think this needs to be better highlighted in the documentaiton.  I did come across examples that put a NULL after it and maybe it exists somewhere in the documentation but I couldn't find it.

**Comment by gaborcsardi** [comment]
Yeah, I suppose it should be here: https://roxygen2.r-lib.org/articles/rd.html#basics

**Comment by ggrothendieck** [comment]
An actual example that uses @import and NULL would be nice.
